examples/bad_example.py

```
@@ -0,0 +1,19 @@
+def run():
+  code = "print('Hello World')"
+  eval(code)
+
+  user_input = input("Enter a number: ")
+  try:
+    result = int(user_input) / 0
+    print(   result  )
+  except:
```

Ruff (0.11.9)

Do not use bare `except`

(E722)